Packs/FraudWatch/Integrations/FraudWatch/FraudWatch.py

```diff
@@ -0,0 +1,854 @@
+        - (None): If argument does not exist, returns None.
+        - (Exception): If argument exists and is lower than min, raises DemistoException.
+    """
+    argument_value = arg_to_number(args.get(argument_name), arg_name=argument_name)
+
+    if argument_value and argument_value < MINIMUM_POSITIVE_VALUE:
+        raise DemistoException(f'{argument_name} should be equal or higher than {MINIMUM_POSITIVE_VALUE}')
+
+    return argument_value
+
+
+def get_time_parameter(arg: Optional[str], parse_format: bool = False):
+    """
+    parses arg into date time object with aware time zone if 'arg' exists.
+    If no time zone is given, sets timezone to UTC.
+    Returns the date time object created or the expected format by FraudWatch.
+    Args:
+        arg (str): The argument to turn into aware date time.
+        parse_format (bool): Should return date or the parsed format of the date.
+
+    Returns:
+        - (None) If 'arg' is None, returns None.
+        - (datetime): If 'arg' is exists and parse_format is false, returns date time.
+        - (str): If 'arg' is exists and parse_format is true, returns the FraudWatch date format of the date time object.
+    """
+    maybe_unaware_date = arg_to_datetime(arg, is_utc=True)
+    if not maybe_unaware_date:
+        return None
+
+    aware_time_date = maybe_unaware_date if maybe_unaware_date.tzinfo else UTC_TIMEZONE.localize(
+        maybe_unaware_date)
+
+    if parse_format:
+        return aware_time_date.strftime('%Y-%m-%d')
+    return aware_time_date
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def fetch_incidents_command(client: Client, params: Dict, last_run: Dict):
+    """
+    Fetches new incidents from FraudWatch.
+    Because of FraudWatch limitation, such as 'from_date' being accurate only by day, and incidents returned
+    are not ordered by their date opened.
+    1) First_fetch time is limited to 1 day.
+    2) Every fetch_incidents call, we will fetch all the pages from the day of the time stamp saved in last run.
+    3) Filter all incidents that their date opened was earlier than last saved time stamp.
+    4) Sort the remained incidents by their date opened.
+    5) Return the first 'max_fetch' incidents in the sorted list.
+    Limiting first_fetch is needed because we must traverse all incidents that might be relevant, therefore the
+    list cannot be very large.
+    Uses Demisto last run parameters in the following way:
+    - 'last_fetch_time' - The latest time fetching was done. Keeps track of incidents to be fetched by their date.
+
+    Args:
+        client (Client): FraudWatch client to perform API call to fetch incidents.
+        params (Dict): Demisto params.
+        last_run (Dict): Demisto last run.
+
+    Returns:
+        (List[Dict], Dict): Incidents and new run parameters.
+    """
+    brand = params.get('brand')
+    status = params.get('status')
+    limit = int(params.get('max_fetch', 50))
```

Comment:
@ShahafBenYakir 
Reason i do this is because i get an error in demisto-sdk lint in the following line
    num_of_incidents_to_fetch = limit if limit <= len(incidents) else len(incidents)
    because arg_to_number returns Optional[int], it said the <= operation is not allowed and fails lint.
    What should I do?